code/modules/events/event.dm

```diff
@@ -41,11 +41,10 @@
 
 	return total_weight
 
-/datum/event_meta/extended_penalty
-	var/penalty = 100 // A simple penalty gives admins the ability to increase the weight to again be part of the random event selection
+/datum/event_meta/extended_removed // will not spawn if gamemode is extended
 
-/datum/event_meta/extended_penalty/get_weight()
-	return ..() - (istype(SSticker.mode, /datum/game_mode/extended) ? penalty : 0)
```

Comment:
???

Reply:
Oh wait I just realized what the intent was here, reverting.